### PR TITLE
Feature: navbar component

### PR DESCRIPTION
**Pull request by daveclinton**
# Add Navbar Component
This change adds the desktop version of the Navbar Component

Fixes # ([issue](https://github.com/SpaceyaTech/Team-Berlin-React/issues/25))
image

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] Tested manually on the browser

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
```
Comment (nyandika) [inline src/components/navbar/index.tsx:1]: I am on the fence over this one. Why are we importing everything? shouldn't you import only the bits you need?

File: src/components/navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          >
+            About Us
+          </a>
+        </div>
+        <div>
+          <button className="bg-transparent  text-secondaryGreen font-semibold  py-2 px-4 border w-32  rounded mr-4">
+            Login
+          </button>
+          <button className="bg-secondaryGreen text-primaryWhite font-semibold  py-2 px-4 border w-32 h-12  rounded">
+            Sign In
```
Comment (nyandika) [inline src/components/navbar/index.tsx]: did you mean sign up instead?

**Review by nyandika — APPROVED** [review]
Just a few nits. otherwise looks good

**Comment by nyandika** [comment]
Forgot a bit: You can use this [link](https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-file) to ensure .eslintcache is being ignored. as is right now it will still be committed

**Comment by daveclinton** [comment]
> Forgot a bit: You can use this [link](https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-file) to ensure .eslintcache is being ignored. as is right now it will still be committed

Alright

File: src/components/navbar/index.tsx (hunk 1 above)
Comment (daveclinton) [reply src/components/navbar/index.tsx:1]: image
It gives me this error when i fail to import. And still I'm going to use the import once i get the design for the mobile version of the navbar

File: src/components/navbar/index.tsx (hunk 2 above)
Comment (daveclinton) [reply src/components/navbar/index.tsx]: I have addressed this change.

**Review by jumalaw98 — APPROVED** [review]

**Pull request merged**